tests/test_blend.py

```diff
@@ -88,16 +90,17 @@ def test_fit_point_source(self):
         blend.fit(10)
 
         assert blend.it == 2
-        assert_almost_equal(blend.L_sed, 2.5481250470053265)
-        assert_almost_equal(blend.L_morph, 9024.538938935855)
+        #assert_almost_equal(blend.L_sed, 2.5481250470053265)
+        #assert_almost_equal(blend.L_morph, 9024.538938935855)
```

Comment:
Remove?

Reply:
Yeah. I left it commented out to leave it open for discussion during review but I agree with Peter's sentiment from a few weeks ago that we should eliminate tests for internal things that the users don't see, and these parameters are so dependent on the internal workings of the code that the tests are essentially useless.